.github/dependabot.yml

```diff
@@ -12,3 +12,9 @@ updates:
   schedule:
     interval: daily
   open-pull-requests-limit: 10
+
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
```

Comment:
Why the PR limit? (I know it was already there still I'm wondering why it's necessary).

Reply:
I guess to limit the PRs it can create at a time, but I can also remove it, I just wanted it to be identical to the other config.

Reply:
> By default, Dependabot opens a maximum of five pull requests for version updates.

– https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit

Reply:
Then 5 PRs a day is more than enough for the GitHub Actions, I would say.

Reply:
> 5 PRs a day

This is not a _rate_ limit:

> Once there are five open pull requests from Dependabot, Dependabot will not open any new requests until some of those open requests are merged or closed.

– https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit

Reply:
Nevertheless, the default limit is more than enough for the GA.